.zuul.yaml

```diff
@@ -35,75 +35,80 @@
-        tempest.api.identity.admin.v3.test_users.UsersV3TestJSON.test_update_user_password
-      cifmw_tempest_tempestconf_config:
-        overrides: |
-          compute-feature-enabled.dhcp_domain ''
-          identity.v3_endpoint_type public
-          service_available.swift false
-          service_available.cinder false
-      cifmw_test_operator_tempest_workflow:
-        - stepName: 'full'
-        - stepName: 'single-test'
-          tempestRun:
-            includeList: |
-              tempest.api.compute.admin.test_flavors.FlavorsAdminTestJSON.test_create_flavor_using_string_ram
-      cifmw_test_operator_tempest_extra_images:
-        - URL: https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img
-          name: cirros-0.6.2-test-operator
-          imageCreationTimeout: 300
-          flavor:
-            name: cirros-0.6.2-test-operator-flavor
-            RAM: 512
-            disk: 20
-            vcpus: 1
-      cifmw_test_operator_tempest_ntp_extra_images: https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img
-      cifmw_test_operator_tempest_network_attachments:
-        - ctlplane
+      cifmw_test_operator_stages:
+        - name: tempest
+          type: tempest
+          test_vars:
+            cifmw_test_operator_tempest_include_list: |
+              ^tempest.
+            cifmw_test_operator_tempest_exclude_list: |
+              # Note (lpiwowar): Unskip test_verify_hostname_allows_fqdn once the
+              #                  Details: Timeout while verifying metadata on server
+              #                  error is resolved.
+              tempest.api.compute.servers.test_create_server.ServersV294TestFqdnHostnames.test_verify_hostname_allows_fqdn
+              tempest.api.compute.admin.test_live_migration.LiveMigrationTest.test_live_migration_with_trunk
+              tempest.api.compute.admin.test_auto_allocate_network.AutoAllocateNetworkTest.test_server_multi_create_auto_allocate
+              tempest.api.compute.servers.test_server_rescue.ServerStableDeviceRescueTestIDE
+              tempest.api.compute.servers.test_device_tagging
+              tempest.api.network.admin.test_dhcp_agent_scheduler.DHCPAgentSchedulersTestJSON
+              tempest.scenario.test_minimum_basic.TestMinimumBasicScenario.test_minimum_basic_scenario
+              tempest.scenario.test_stamp_pattern
+              tempest.scenario.test_network_advanced_server_ops.TestNetworkAdvancedServerOps.test_server_connectivity_live_migration
+              tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_mtu_sized_frames
+              test_create_server_invalid_bdm_in_2nd_dict
+              tempest.api.identity.admin.v3.test_credentials.CredentialsTestJSON
+              tempest.api.identity.admin.v3.test_tokens.TokensV3TestJSON.test_rescope_token
+              tempest.api.identity.admin.v3.test_users.UsersV3TestJSON.test_update_user_password
```

Comment:
@eshulman2 this is a correct job definition that should work with the change in the test-operator role, right?

It seems like the include_list and exclude_list values did not get propagate correctly? -> https://logserver.rdoproject.org/15/215/824b0bc5fcba970c32b8f5b2ffab3f368e2b0940/github-check/podified-multinode-edpm-deployment-crc-test-operator/909305c/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/tempests.test.openstack.org/tempest-tests-tempest.yaml

Reply:
should be resolved now